### DISCUSSION TITLE
Trailing Commas in CTEs

### DISCUSSION THREAD

**Discussion by Admolly** [duckdb/duckdb#3983] [Ideas]
An idea popped into my head while I was reading the "Friendlier SQL" blog post.

Would it be possible to do the same thing for CTEs?  I've always found it quite irritating when I delete or comment out the last CTE in query and get the trailing comma error.  Would be super helpful!

**Comment by paultiq** [comment]
+1

This comes up constantly for me, as I build a query starting with one or two CTEs, it's convenient to place a trailing comma after each CTE.

Trailing commas are also diff friendlier, consistent with other elements, and makes CTE construction easier.
